packages/search/addon/services/navi-search-provider.js

```diff
@@ -0,0 +1,29 @@
+import Service from '@ember/service';
+import { getOwner } from '@ember/application';
+import config from 'ember-get-config';
+
+/* global requirejs */
+
+export default class NaviSearchProviderService extends Service {
+  /**
+   * @method getService
```

Comment:
maybe `getProvider`

Reply:
or `getSearchProviderByName`

Reply:
or `getById`

Reply:
`getByName`

Reply:
when others use this service they'd do something like: 
`searchProviderService: service.inject()`

`searchProviderService.all()`
or
`searchProviderService.getById()`

Reply:
I think we can do `getByName` since in the search provider we have a name property `name = 'reportSearchProvider';`